contrib/debian/control

```diff
@@ -53,7 +53,8 @@ Conflicts: netdata-core,
            netdata-plugins-python,
            netdata-web
 Suggests: netdata-plugin-cups,
-          netdata-plugin-freeipmi
+          netdata-plugin-freeipmi,
+          netdata-plugin-systemd-journal
```

Comment:
If I remember correctly, this will _not_ cause the newly suggested dependency to be installed when upgrading existing installs. Is there something we can do to trigger additional suggests on upgrades?

Reply:
We intentionally _do not_ want this one pulled in by default. Debian users who have chosen to use sysvinit instead of systemd will be exceedingly grumpy if we do, both because they don’t need it and because it will likely pull in a huge tree of dependencies for them.

Reply:
So, to satisfy @ktsaou's requirement for always having this plugin installed when systemd is installed on the system, this dependency needs to move to `Depends:`.